sensing/autoware_pointcloud_preprocessor/src/outlier_filter/ring_outlier_filter_node.cpp

```diff
@@ -47,22 +47,19 @@ RingOutlierFilterComponent::RingOutlierFilterComponent(const rclcpp::NodeOptions
 
   // set initial parameters
   {
-    distance_ratio_ = static_cast<double>(declare_parameter("distance_ratio", 1.03));
-    object_length_threshold_ =
-      static_cast<double>(declare_parameter("object_length_threshold", 0.1));
-    num_points_threshold_ = static_cast<int>(declare_parameter("num_points_threshold", 4));
-    max_rings_num_ = static_cast<uint16_t>(declare_parameter("max_rings_num", 128));
-    max_points_num_per_ring_ =
-      static_cast<size_t>(declare_parameter("max_points_num_per_ring", 4000));
-    publish_outlier_pointcloud_ =
-      static_cast<bool>(declare_parameter("publish_outlier_pointcloud", false));
-
-    min_azimuth_deg_ = static_cast<float>(declare_parameter("min_azimuth_deg", 0.0));
-    max_azimuth_deg_ = static_cast<float>(declare_parameter("max_azimuth_deg", 360.0));
-    max_distance_ = static_cast<float>(declare_parameter("max_distance", 12.0));
-    vertical_bins_ = static_cast<int>(declare_parameter("vertical_bins", 128));
-    horizontal_bins_ = static_cast<int>(declare_parameter("horizontal_bins", 36));
-    noise_threshold_ = static_cast<int>(declare_parameter("noise_threshold", 2));
+    distance_ratio_ = declare_parameter<double>("distance_ratio");
+    object_length_threshold_ = declare_parameter<double>("object_length_threshold");
+    num_points_threshold_ = declare_parameter<int>("num_points_threshold");
+    max_rings_num_ = declare_parameter<int64_t>("max_rings_num");
```

Comment:
inconsistency with the variable type (missing cast)

Reply:
fixed in https://github.com/autowarefoundation/autoware.universe/pull/8468/commits/21dc9d4f8e3b665bf42ece734d3a63d00f6d0d64